File: src/Processing/MATLAB/call_preprocess.m
%% Ocean Engineering Toolbox - Calling the Pre-Processing Function
%{
A script calling the post-processing function for the OET to pre-process 
hydrodynamic data. The user must call this function in a script within the 
same folder as the .h5, enter the current file pathfile, and specify the 
device name.
%}

currentPath = pwd;
filePath = 'deviceName.h5';
deviceName = 'deviceName';


[hydro] = preprocess(currentPath, filePath, deviceName);

File: src/Processing/Python/Preprocess/call_preprocess.py
"""
Ocean Engineering Toolbox - Calling the Pre-Processing Function (Python)
"""
import os
from preprocess import preprocess

def main():
    current_path = os.getcwd()          # equivalent to MATLAB pwd
    file_path = 'rm3.h5'                # .h5 filename in the same folder
    device_name = 'rm3'                 # prefix for saved mat file

    hydro = preprocess(current_path, file_path, device_name)

if __name__ == "__main__":
    main()